lib/chef/handler/datadog_chef_tags.rb

```diff
@@ -92,6 +92,7 @@ def with_policy_tags_enabled(enabled)
   def send_update_to_datadog(dog)
     tags = combined_host_tags
     retries = @retries
+    rc = []
```

Comment:
You should use `rc.inspect` when using this variable at L118 because the string representation of an array is a the string representation of its elements separated by `\n`. With inspect, everything will be kept in the line.